README.md

```diff
@@ -44,12 +44,16 @@ or
 **Documentation**: [current release](https://3yourmind.github.io/kotti/) or [current master](https://kotti.netlify.com/)
 
 ```typescript
-// in main.js / entrypoint
+// in main.ts / entrypoint
```

Comment:
arguably we should just say in your entry point, no specification of a ts must or a js must
and maybe no specification of `main` altho it's ofc the default if we spin up a vue project

or what do u think

Reply:
But it says `/ entrypoint`?

Reply:
yes im retarded :)